citi-onboarding/sustmat-one-page
# Feature/advantagesfront

### What I did:
- Created the advantages section

### How to test:
- git checkout develop
- git pull
- git checkout feature/advantagesfront
- yarn start 
- test and see if it works on the local server
